.github/workflows/reminders.yml

```diff
@@ -0,0 +1,16 @@
+name: PRs reviews reminder
+
+on:
+  schedule:
+    # Every weekday every 2 hours during working hours, send notification
+    - cron: "* * * * *"
+
+jobs:
+  pr-reviews-reminder:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: davideviolante/pr-reviews-reminder-action@microsoftteams
```

Comment:
Can we freeze this to the current version of the repository (the one I checked the code for) - so that any new changes don't affect us until we intentionally update?